config/config.go

```diff
@@ -24,13 +29,19 @@ func DefaultConfig() *Config {
 			ListenAddr:     "127.0.0.1:8080",
 		},
 		Panacea: PanaceaConfig{
-			GRPCAddr: "127.0.0.1:9090",
+			ChainID:          "",
+			GRPCAddr:         "127.0.0.1:9090",
+			DefaultGasLimit:  300000,
+			DefaultFeeAmount: "1000000umed",
```

Comment:
Would be set `DefaultGasLimit` to `200000` if you're gonna set `DefaultFeeAmount` to `1000000umed`, because our recommended gas price is `5umed`.

Reply:
Right. I changed `DefaultGasLimit` to 200000 in [5a20456](https://github.com/medibloc/panacea-doracle/pull/11/commits/5a2045637c4e4834e5cf21efb76488f73152fd0c)